projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -575,6 +574,18 @@ public String getAnswer(
       return answer;
    }
 
+   /**
+    * Return a {@link SortedSet SortedSet} contains all subdirectories under the path
+    * {@code containerDir}
+    */
+   public SortedSet<String> getContainer(Path containerDir) {
+      SortedSet<String> testrigs = new TreeSet<>(
+            CommonUtil.getSubdirectories(containerDir).stream()
+                  .map(dir -> dir.getFileName().toString())
+                  .collect(Collectors.toSet()));
```

Comment:
Technically, testrigs do not sit directly under the container directory. They sit under <containerdir>/BfConsts.RELPATH_TESTRIGS_DIR. (The constant is "" right now for backward compatibility but we'll make it something else at some point.)